include/revng-c/RestructureCFG/GenericRegionPass.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+//
+// Copyright rev.ng Labs Srl. See LICENSE.md for details.
+//
+
+#include "llvm/IR/Function.h"
+#include "llvm/Pass.h"
+
+class GenericRegionPass : public llvm::FunctionPass {
+private:
+public:
+  static char ID;
+
+public:
+  GenericRegionPass() : llvm::FunctionPass(ID) {}
+
+  bool runOnFunction(llvm::Function &F) override;
+
+  void getAnalysisUsage(llvm::AnalysisUsage &AU) const override;
+
+  void getResult();
```

Comment:
Yeah, this slipped away, since I'm running the unit tests with `FileCheck` and not consuming the results, I did not see this, fixed this here https://github.com/revng/revng-c/blob/2874031652f8d9c4240ed2796912c23c75054aca/include/revng-c/RestructureCFG/GenericRegionPass.h#L26